### ISSUE TITLE
[somfytahoma] Triggering alarm check of smoke sensors not working as documented

### ISSUE THREAD

**Issue by hypetsch** [openhab/openhab-addons#12858]
Recently I tried to trigger the short and long checks of my Somfy IO smoke sensors using the "alarm check trigger channel".

As mentioned in the [docs](https://github.com/openhab/openhab-addons/blob/598d88648c69f356df16cebe2ceae33282864dbd/bundles/org.openhab.binding.somfytahoma/README.md?plain=1#L91) I tried "short_check" and "long_check" - both did not work:
```
events.sendCommand("RauchmelderSchlafzimmer_AlarmCheckTrigger", "short_check");
```

After that I monitored the traffic sent when triggering the check from the Tahoma web UI and found out that they use a different parameter (`short` vs. `short_check`):
```
{
    "startTime": 1653980107373,
    "owner": "xxxxxx@gmx.at",
    "actionGroup": {
        "label": "RauchSchlafzimmer - Testen - TaHoma PC",
        "shortcut": false,
        "notificationTypeMask": 0,
        "notificationCondition": "NEVER",
        "actions": [
            {
                "deviceURL": "io://1234-5678-9999/123465789",
                "commands": [
                    {
                        "type": 1,
                        "name": "checkEventTrigger",
                        "parameters": [
                            "short"
                        ]
                    }
                ]
            }
        ]
    },
    "description": "Execution : RauchSchlafzimmer - Testen - TaHoma PC",
    "id": "12345678-ac10-3e01-3a06-3813651e9b06",
    "state": "TRANSMITTED",
    "executionType": "Immediate execution",
    "executionSubType": "MANUAL_CONTROL"
}
```

The check can successfully be triggered with 
```
events.sendCommand("RauchmelderSchlafzimmer_AlarmCheckTrigger", "short");
events.sendCommand("RauchmelderSchlafzimmer_AlarmCheckTrigger", "long");
```

So I am not sure whether this is just a glitch in the documentation or the interface has changed on Somfy side - just wanted to let you know.

**Comment by octa22** [comment]
Yeah most probably forgotten doc update. Should be fixed in https://github.com/openhab/openhab-addons/pull/13919

**Comment by lolodomo** [comment]
Closed by #13919
